src/test/java/bio/terra/workspace/integration/common/utils/WorkspaceManagerTestClient.java

```diff
@@ -0,0 +1,81 @@
+package bio.terra.workspace.integration.common.utils;
+
+import bio.terra.workspace.generated.model.ErrorReport;
+import bio.terra.workspace.integration.common.auth.AuthService;
+import bio.terra.workspace.integration.common.response.ObjectOrErrorResponse;
+import bio.terra.workspace.integration.common.response.WorkspaceResponse;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.*;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+@Component
+public class WorkspaceManagerTestClient {
```

Comment:
can/should the integration tests use the workspace manager client lib, instead of writing a custom client?

Reply:
Where is this `workspace manager client lib`? Is it an artifact that's importable with gradle? I don't know if I'm looking at the right places.

Reply:
See https://github.com/broadinstitute/rawls/pull/1197/commits/75036f727e2a82b17e2d80f3fc57bd19f47070d2, in broadinstitute/rawls#1197, for example code

Reply:
Discussed in Slack. Keeping raw HTTP requests.